gulpfile.js

```diff
@@ -113,8 +113,7 @@ function watch(done) {
     livereload: true
   });
 
-  mainWatcher.on('all', gulp.series(clean, gulp.parallel(lint, 'build-bundle-dev', test)));
-  loaderWatcher.on('all', gulp.series(lint));
+  mainWatcher.on('all', gulp.series(clean, gulp.parallel('build-bundle-dev')));
```

Comment:
Same with the changes here, will remove, as are for testing only and accidentally checked in